tasks/terraform-cli/src/tests/steps/task-context.steps.ts

```diff
@@ -160,6 +161,27 @@ export class TaskContextSteps {
         this.ctx.skipExistingWorkspace = true;
     }
 
+    @given("subcommand is {string}")

```

Comment:
can we make this state specific?
```suggestion
    @given("state subcommand is {string}")
```